dev-docs/source/AlgorithmMPISupport.rst

```diff
@@ -1,640 +0,0 @@
-- If the detectors are moveable, a spectrum would correspond to a region of space given by the position a specific detector had in a certain time interval.
-- There can be detectors without a corresponding spectrum, and Mantid is thus not handling data of those detectors.
-
-For the purpose of MPI support, Mantid always stores the complete instrument including **all detectors** on **every MPI rank**. [#split-instrument]_
-Only a subset of all spectra is stored on the local MPI rank, so a detector may be locally without corresponding spectrum but have a spectrum associated to it on another MPI rank.
-
-
-Storage Mode
-------------
-
-.. figure:: images/MPI-storage-modes.png
-   :figwidth: 25%
-   :align: right
-
-   Visualization of available storage modes. In this and the following figures a vertical column is used to depict an MPI rank.
-
-Not all data in an MPI-based data reduction can be or has to be distributed.
-Some data may be present on all ranks or only on a single rank.
-To formalize this concept a ``Workspace`` in Mantid now has an associated storage mode (``Parallel::StorageMode``), as visualized in the figure on the right.
-The storage mode of a workspace can be obtained by calling ``Workspace::storageMode()``.
-There are three available storage modes
-
-- ``StorageMode::Cloned`` implies that the data is not split and each rank holds a complete and identical clone.
-- ``StorageMode::Distributed`` implies that each rank holds a subset of the data and that the combination of all subsets gives the full data.
-- ``StorageMode::MasterOnly`` implies that only the master rank (rank 0) has the data.
-
-Currently only workspaces have an associated storage mode.
-For all other data, such as a scalar value provided as an input to an algorithm are implicitly assumed to have ``StorageMode::Cloned``, i.e., have the same value on all MPI ranks.
-
-Usage examples for the storage modes could include:
-
-- A workspace containing the neutron monitors could be used for normalization on all ranks and could use ``StorageMode::Cloned``.
-- A workspace containing the data for all detector pixels would usually use ``StorageMode::Distributed``.
-- A workspace containing the result of summing all spectra, such as obtained from ``DiffractionFocussing`` would usually use ``StorageMode::MasterOnly``.
-
-Execution Mode
---------------
-
-.. figure:: images/MPI-execution-mode-identical.png
-   :figwidth: 25%
-   :align: right
-
-   ``ExecutionMode::Identical`` based on an input and output workspace with ``StorageMode::Cloned``. Example: ``ConvertUnits``, ``Rebin``, or many other algorithm that do not load or save data.
-
-
-.. figure:: images/MPI-execution-mode-distributed-load.png
-   :figwidth: 25%
-   :align: right
-
-   ``ExecutionMode::Distributed`` creating an output workspace with ``StorageMode::Distributed``. Example: ``LoadEventNexus``.
-
-.. figure:: images/MPI-execution-mode-distributed.png
```

Comment:
Those images can also be remove, I presume?

Reply:
ok